src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -205,6 +205,10 @@ public Album GetLocalAlbum(string filename, Artist artist)
 
             filename = artist.Path.GetRelativePath(filename);
 
+            // Joe: We can clean here
+            filename = Parser.CleanAlbumTitle(filename);
+            _logger.Debug("GetLocalAlbum, cleaned album title: {0}", filename);
```

Comment:
I think adding this to this method will mess up filename matching for extras since you are cleaning the actual filename that its searching the db for. I would remove from `GetLocalAlbum`

Reply:
Hmmm good point, I didn't test against extras. Let me test moving this just to one location and test it out.